bundles/config/org.eclipse.smarthome.config.discovery/src/main/java/org/eclipse/smarthome/config/discovery/internal/DiscoveryResultImpl.java

```diff
@@ -215,7 +228,16 @@ public boolean equals(Object obj) {
     @Override
     public String toString() {
         return "DiscoveryResult [thingUID=" + thingUID + ", properties=" + properties + ", flag=" + flag + ", label="
-                + label + ", bridgeUID=" + bridgeUID + "]";
+                + label + ", bridgeUID=" + bridgeUID + ", ttl=" + timeToLive + ", createdAt=" + createdAt + "]";
+    }
+    
+    @Override
+    public long getCreatedAt() {
```

Comment:
Well, naming a variable "timestamp" would be the same as naming it "integer" or "boolean" or "string". It says nothing about the meaning of this variable. So what kind of timestamp is it? This was the reason why I suggested to give it another name. I know it is already merged, but we should avoid to name public methods or variables like this.